.github/workflows/Nightly.yml

```diff
@@ -4,6 +4,8 @@ on:
     inputs:
       cmake_args:
         description: 'Extra CMake args'
+      ctest_args:
```

Comment:
Needs to be rebased

Reply:
done

Reply:
Need to be careful when you do that - my last commits were overwritten

Reply:
sorry, anything to be fixed?